library/globals.inc.php

```diff
@@ -3459,6 +3459,13 @@ function gblTimeZones()
             xl('Each provider needs to set this under user settings. This should be blank')
         ),
 
+        'weno_setup_instructions_link' => array(
+            xl('Weno Setup instructions for OpenEMR'),
+            'text',                      // data type
+            '',
+            xl('Check yoour Weno Developer page for instructions')
```

Comment:
mispelled `yoour`

Reply:
Does this need to be something or gets populated after turning on weno?

Reply:
Will correct the spelling.

Ideally, it should be pre-populated with the URL for Weno Developer page but the CEO of Weno, suggested that the field should be left empty because the link may change. I could take it off if you do not see any need for it